.gitignore

```diff
@@ -31,6 +31,9 @@ cache/
 /bin/z3-x64.exe
 /src/*/obj
 
+# Generated by build
+bin/fstar-compiler-lib
+
```

Comment:
When I do `git status`, I don't see any files in `bin` in the output, why do we need this entry in `.gitignore`?

Reply:
Honestly I forget on what exact configuration it was produced, maybe that's me fiddling, Do I need to provide you with repro steps on clear repo how I reach that state?

Reply:
That would be great to know, thanks!